### PR TITLE
Create SP19-CulturalHeritageSites.md

### PR DESCRIPTION
**Pull request by mjanas**

### CODE REVIEW

File: _projects/SP19-CulturalHeritageSites.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: Cultural Heritage Sites (Cahokia Mounds)
+image:
+desc: Virtual experiene of Cahokia Mounds, with the environment created using LiDAR data.
+github:
+dl: https://drive.google.com/drive/folders/1dZ67RJ7ryCFAwCrcGxavylECdB9qpkZ2?usp=sharing
```
Comment (ShanTulshi) [inline _projects/SP19-CulturalHeritageSites.md]: The drive folder is missing a Data folder, needed to run the project

File: _projects/SP19-CulturalHeritageSites.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: Cultural Heritage Sites (Cahokia Mounds)
+image:
+desc: Virtual experiene of Cahokia Mounds, with the environment created using LiDAR data.
+github:
```
Comment (ShanTulshi) [inline _projects/SP19-CulturalHeritageSites.md]: please remove this line if you are not using it
```suggestion
```

File: _projects/SP19-CulturalHeritageSites.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: Cultural Heritage Sites (Cahokia Mounds)
+image:
```
Comment (ShanTulshi) [inline _projects/SP19-CulturalHeritageSites.md]:
```suggestion
```

File: _projects/SP19-CulturalHeritageSites.md (hunk 1 above)
Comment (mjanas) [reply _projects/SP19-CulturalHeritageSites.md]: Added Data folder and other stuff just in case

**Pull request merged**